sdformat_mjcf_utils/sdformat_mjcf_utils/sdf_utils.py

```diff
@@ -41,6 +51,17 @@ def vec2d_to_list(vec):
     return [vec.x(), vec.y()]
 
 
+def list_to_quat(list):
```

Comment:
Consider renaming to `euler_list_to_quat` or something similar that indicates 
the input is euler angles.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/21b76c3da90666223352811af70d216f8f722e68